examples/glamorous-jest-react/README.md

```diff
@@ -0,0 +1,9 @@
+## Example for `glamorous` with `jest` and `enzyme`
+
+### Steps to run
```

Comment:
Could you add `git clone` instructions as well. Want to make this as beginner-friendly as possible :)